src/app/cli/src/graphql.ml

```diff
@@ -45,17 +45,20 @@ module Types = struct
 
     let amount amount = uint64 @@ Currency.Amount.to_uint64 amount
 
-    module State_hash = Codable.Make_base64 (State_hash.Stable.V1)
+    module State_hash = Codable.Make_base58_check (State_hash.Stable.V1)
   end
 
   module Id = struct
-    (* The id of a user_command is base64 the seralized version of the user_command *)
+    let version_byte = '\x17'
```

Comment:
How are all these `version_byte` values chosen? Should we gather them in one variant somewhere so we can easily tell that they are all distinct?

Reply:
Chosen randomly. We can create a file of them in `base58_check`, if you'd like.

Reply:
Yes that seems like a good idea

Reply:
Done.